src/extlibs/er_force_sim/src/protobuf/ssl_simulation_custom_erforce_realism.proto

```diff
@@ -4,6 +4,7 @@ option go_package = "github.com/RoboCup-SSL/ssl-simulation-protocol/pkg/sim";
 // Realism configuration for ER-Force simulator
 // this message is stored in human readable form for saved ui settings
 // therefore do NOT modify field names without changing them in the config files
+// the values are set to 0 to simulate the ideal situation
```

Comment:
nit: I don't think we need this comment, since in reality we set some of the values to something other than 0 for the ideal scenario